yb-voyager/src/migassessment/assessmentDB.go

```diff
@@ -146,6 +147,15 @@ func InitAssessmentDB() error {
 			redundant_ddl TEXT,
 			existing_ddl TEXT,
 			PRIMARY KEY(redundant_schema_name,redundant_table_name,redundant_index_name));`, REDUNDANT_INDEXES),
+		fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s (
+			schema_name TEXT,
+			table_name TEXT,
+			column_name TEXT,
+			null_frequency REAL,
+			effective_n_distinct INTEGER,
+			max_frequency REAL,
+			max_frequent_val TEXT,
```

Comment:
shall we call this `most_common_freq` and `most_common_val` ? In the pg stats table, these values are arrays, but in our case, as I understand we're only picking the first one, so it's still the "most" common val/freq.